CSharp/Library/Microsoft.Bot.Builder/Resource/Resources.resx

```diff
@@ -356,4 +356,19 @@
   <data name="UserProfileDeleted" xml:space="preserve">
     <value>User profile deleted!</value>
   </data>
+  <data name="NumberExpression" xml:space="preserve">

```

Comment:
these need to be added to xlf files so resources can be generated by multilingual app toolkit: [https://developer.microsoft.com/en-us/windows/develop/multilingual-app-toolkit](https://developer.microsoft.com/en-us/windows/develop/multilingual-app-toolkit)